draft-wqb-tvr-applicability.md

```diff
@@ -69,9 +69,10 @@ informative:
 
 --- abstract
 
-This document describes the applicability of various network management tools
-and data models that may be used for scheduling, to meet the requirements of a set
-of representative use cases described in the "TVR (Time-Variant Routing) Requirements" (I-D.ietf-tvr-requirements).
+This document provides the applicability statement on how the TVR(Time-Variant Routing)
+data model may be used for scheduling in specific time variant network cases, to meet
+the requirements of a set of representative use cases described in the
+"TVR Requirements" (I-D.ietf-tvr-requirements).
```

Comment:
```suggestion
"TVR (Time-Variant Routing) Requirements" (I-D.ietf-tvr-requirements).
```
I think it should be a faithful copy of the title of https://datatracker.ietf.org/doc/draft-ietf-tvr-requirements/.